src/core/tests/frontend/paddle/test_models/gen_scripts/generate_sum.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import print_function
+import sys
+import random
+import paddle
+import paddle.fluid as fluid
```

Comment:
Can you please avoid using fluid api and use paddle 2.1 api instead? Fluid is obsolete since 1.8.

Reply:
done, change `paddle.fluid.layer.sum` to `paddle.add_n`